templates/admin/plugins/persona.tpl

```diff
@@ -0,0 +1,23 @@
+<div class="row">
+	<div class="col-sm-2 col-xs-12 settings-header">Theme Settings</div>
+	<div class="col-sm-10 col-xs-12">	
+		<form>
+			<div class="checkbox">
+				<label class="mdl-switch mdl-js-switch mdl-js-ripple-effect">
+					<input class="mdl-switch__input" type="checkbox" data-field="hideSubCategories">
+					<span class="mdl-switch__label"><strong>Hide subcategories on categories view</strong></span>
+				</label>
+			</div>
+		</form>
+	</div>
+</div>
+	
+<button id="save" class="floating-button mdl-button mdl-js-button mdl-button--fab mdl-js-ripple-effect mdl-button--colored">
+    <i class="material-icons">save</i>
+</button>
+
+<script>
+	require(['admin/settings'], function(Settings) {
+		Settings.prepare();
+	});
+</script>
```

Comment:
Inline js :rage2: 

[Do this instead](https://github.com/NodeBB/nodebb-plugin-quickstart/blob/master/static/lib/admin.js)